src/coreclr/vm/jitinterface.cpp

```diff
@@ -7476,6 +7476,47 @@ class MethodInfoHelperContext final
     }
 };
 
+enum class TransientCodeKind
+{
+    None = 0,
+
+    // Scenario involving C++/CLI copy constructor scenario.
+    CopyConstructor,
+};
+
+static bool IsMarkedTransientMethod(MethodDesc* pMD, TransientCodeKind& kind)
+{
+    STANDARD_VM_CONTRACT;
+    _ASSERTE(pMD != NULL);
+    _ASSERTE(pMD->IsIL());
+    _ASSERTE(pMD->GetRVA() == 0);
+
+    // Transient code paths are currently limited to C++/CLI scenarios.
+    // This can be changed when new non-Windows options are added.
+#ifdef TARGET_WINDOWS
```

Comment:
Use the IJW feature flag instead of "targets Windows"? Not sure if the corresponding flag exists for the CoreLib build.

```suggestion
#ifdef FEATURE_IJW
```